test/models/rsvp_test.rb

```
@@ -0,0 +1,32 @@
+require "test_helper"
+
+class RsvpTest < ActiveSupport::TestCase
+  def setup
+    @user = User.create!(email: "rsvp_test_#{Time.now.to_i}@example.com", display_name: "Rsvp Tester", password: "password")
```

RuboCop (1.75.5)

[convention] Prefer single-quoted strings when you don't need string interpolation or special symbols.

(Style/StringLiterals)